package.json

```diff
@@ -15,9 +15,9 @@
     "build:ts": "bili",
     "build:api_docs": "rm -rf docs/api && typedoc --plugin typedoc-plugin-markdown --hideBreadcrumbs true",
     "build:contracts": "npm run build:typechain:registry && npm run build:typechain:offerableIdentity && npm run build:typechain:identityManager",
-    "build:typechain:registry": "typechain --target ethers-v4 --outDir ethers './node_modules/@ensdomains/ens/build/contracts/ENSRegistry.json'",
-    "build:typechain:offerableIdentity": "typechain --target ethers-v4 --outDir ethers './node_modules/@ew-did-registry/proxyidentity/build/contracts/OfferableIdentity.json'",
-    "build:typechain:identityManager": "typechain --target ethers-v4 --outDir ethers './node_modules/@ew-did-registry/proxyidentity/build/contracts/IdentityManager.json'",
+    "build:typechain:registry": "typechain --target ethers-v4 --outDir ethers \"./node_modules/@ensdomains/ens/build/contracts/ENSRegistry.json\"",
+    "build:typechain:offerableIdentity": "typechain --target ethers-v4 --outDir ethers \"./node_modules/@ew-did-registry/proxyidentity/build/contracts/OfferableIdentity.json\"",
+    "build:typechain:identityManager": "typechain --target ethers-v4 --outDir ethers \"./node_modules/@ew-did-registry/proxyidentity/build/contracts/IdentityManager.json\"",
```

Comment:
Nice fix for windows 👍 . In general though, probably would be good to have a comment in the commit about the rationale. Maybe amend your commit or even better would be to separate it into two commits, once for the windows fix and one for the `rpcUrl` rename

Reply:
sure no problem.